src/theme/main.less

```
@@ -1,20 +1,62 @@
-// Separator Block
+// //Separator Block
 .block.separator {
+  top: unset;
   display: block;
   height: 54px;
   border: none;
-  margin: 32px 0;
+  margin: 50px 0;
+  clear: both;
+
+  &:not(.inner):not([role='presentation']) {
+    padding: 25px 0 !important;
+  }
+
+  &::after {
+    display: none !important;
+  }
 
   .line {
-    height: 54px;
+    width: unset;
+    height: unset;
 
-    &::after {
+    &:not(.inner)::after {
```

The `.line` element never has a `inner` class, so this doesn't do anything.
```suggestion
    &::after {
```